### PR TITLE
Rename Public-Facing API Methods [minor]

### PR DESCRIPTION
**Pull request by ric-evans**
Newly named methods include: `open_pub()`, `open_sub()`, `open_sub_one()`

closes https://github.com/WIPACrepo/MQClient/issues/33

### CODE REVIEW

File: mqclient/abstract_backend_tests/integrate_queue.py [hunk 1]
```diff
@@ -89,16 +89,18 @@ async def test_12(self, queue_name: str) -> None:
...
+            await p.send(DATA_LIST[0])
             _log_send(DATA_LIST[0])
 
         with pytest.raises(Exception) as excinfo:
-            async with Queue(self.backend, name=f"{queue_name}-fail").recv_one() as d:
+            async with Queue(
+                self.backend, name=f"{queue_name}-fail"
+            ).open_sub_one() as d:
                 all_recvd.append(_log_recv(d))
             assert "No message available" in str(excinfo.value)
```
Comment (dsschult) [inline mqclient/abstract_backend_tests/integrate_queue.py:101]: Just a question: does this assert actually get called?  Or does it need to be un-indented?

**Review by dsschult — COMMENTED** [review]
General question: is there a test where the same Queue instance is used for both `open_pub` and `open_sub`?  That should be possible, right?

File: mqclient/abstract_backend_tests/integrate_queue.py (hunk 1 above)
Comment (ric-evans) [reply mqclient/abstract_backend_tests/integrate_queue.py:100]: that indentation is nasty, but the code flow is correct. I'll make it more clear

**Comment by ric-evans** [comment]
> General question: is there a test where the same Queue instance is used for both `open_pub` and `open_sub`? That should be possible, right?

Yes, integration `test_10`

**Review by dsschult — APPROVED** [review]

**Pull request merged**